src/window.rs

```diff
@@ -95,6 +114,120 @@ fn main() {
     }
 }
 
+struct View {
```

Comment:
`View` should probably get its own module.

Reply:
I think you are right, it has gotten quite big now.
Ideally I would also like the `View` struct to take care of the `EventLoop` to remove mess.